src/utils/AxiosInstance.ts

```diff
@@ -17,6 +17,7 @@ instance.interceptors.request.use(
 
     const kcToken = sessionStorage.getItem(SessionStorageKeys.KeyCloakToken)
     request.headers.common['Authorization'] = `Bearer ${kcToken}`
+    request.headers.common['App-Name'] = import.meta.env.APP_NAME
```

Comment:
Sample call to Auth API:
![image](https://github.com/user-attachments/assets/42f30a67-93c5-4a0d-af83-d38f5e27969b)

Sample call to Legal API:
![image](https://github.com/user-attachments/assets/baaeb3f7-b28e-4772-97e9-28fa740e085f)